draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1356,11 +1358,12 @@ Output:
 
 Steps:
 1. m' = HKDF-Extract(DST, msg)
-2. for i in (1, ..., m):
-3.   info = "H2C" || I2OSP(ctr, 1) || I2OSP(i, 1)
-4.   t = HKDF-Expand(m', info, L)
-5.   e_i = OS2IP(t) mod p
-6. return u = (e_1, ..., e_m)
+2. info' = "H2C" || I2OSP(ctr, 1)       // "H2C" is an ASCII literal
```

Comment:
We should probably avoid using "var prime" as a variable name. It's not representable in most programming languages, and easy to misread.

Could this be "info_prefix" instead?

Reply:
( I accept this is maybe not in scope for this PR)

Reply:
This is a great point. Totally easy to fix here (and `m'`, too).